static/app/components/events/interfaces/spans/utils.spec.tsx

```diff
@@ -1,4 +1,7 @@
-import {getFormattedTimeRangeWithLeadingZero} from './utils';
```

Comment:
merged the tests from tests/js/spec/components/events/interfaces/spans/utils.spec.tsx